docs/patterns/single-new-eks-observability-accelerators/single-new-eks-cost-monitoring-ingress-observability.md

```diff
@@ -0,0 +1,151 @@
+
+## Architecture
+
+The following figure illustrates the architecture of the pattern we will be deploying for Single EKS cost monitoring (Kubecost) pattern with Application Load Balancer, Amazon Cognito, and a Transport Layer Security (TLS) Certificate on AWS Certificate Manager (ACM) with Amazon Route 53 hosted zone to authenticate users to Kubecost
+
+![Architecture](../images/costmonitoring-ingress.png)
+
+## Objective
+
+- Deploys one production grade Amazon EKS cluster.
+- AWS Kubecost with Amazon Managed Prometheus (AMP) integration
+- [Secure Ingress with AWS Cognito](https://aws.amazon.com/blogs/containers/securing-kubecost-access-with-amazon-cognito/)
+- AWS Certificate Manager with Amazon Route 53 hosted zone 
+- Alerts and recording rules with Amazon Managed Service for Prometheus
+
+## Prerequisites:
+
+An existing hosted zone in Route53 with the ability to add records.
+
+Ensure that you have installed the following tools on your machine.
+
+1. [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+2. [kubectl](https://Kubernetes.io/docs/tasks/tools/)
+3. [cdk](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html#getting_started_install)
+4. [npm](https://docs.npmjs.com/cli/v8/commands/npm-install)
+
+## Configuring domain
+
+The CDK code expects the allowed domain and subdomain names in the CDK context file (cdk.json).
+
+Create two environment variables. The PARENT_HOSTED_ZONE variable contains the name of your Route 53 public hosted zone. The DEV_SUBZONE_NAME will be the address for your Kubecost dashboard.
+
+When users register to cognito they will have to provide an email address, using the `allowed.domains.list` you can specify you enterprise's email domain to only allow your employees to sign up for the service
+
+Generate the cdk.json file:
+
+```bash
+PARENT_HOSTED_ZONE=mycompany.a2z.com
+DEV_SUBZONE_NAME=kubecost.mycompany.a2z.com
+ALLOWED_DOMAIN_LIST=amazon.com
+cat << EOF > cdk.json
+{
+    "app": "npx ts-node dist/lib/common/default-main.js",
+    "context": {
+        "parent.hostedzone.name": "${PARENT_HOSTED_ZONE}",
+        "dev.subzone.name": "${DEV_SUBZONE_NAME}",
+        "allowed.domains.list": "${ALLOWED_DOMAIN_LIST}"
+      }
+}
+EOF
+```
+
+
+## Deploying
+
+Please follow the _Deploying_ instructions of the [New EKS Cluster Open Source Observability Accelerator](./single-new-eks-opensource-observability.md) pattern till step 7.
+At step 8, execute the following
+
+```bash
+make build
+make pattern single-new-eks-cost-monitoring deploy single-new-eks-cost-monitoring-observability-accelerator
```

Comment:
The command is not correct. It should be `make pattern single-new-eks-cost-monitoring-observability-accelerator deploy` isnt it?

Reply:
No since there are 2 components to deploy, this is the correct command, it automatically deploys the other dependency as well i.e. Cognito Stack